.github/workflows/release.yml

```diff
@@ -14,24 +17,34 @@ jobs:
         uses: actions/checkout@v2
       
       - name: Log in to Docker Hub
+        if: github.event_name != 'pull_request'
         uses: docker/login-action@v1
         with:
           username: ${{ secrets.DOCKER_USERNAME }}
           password: ${{ secrets.DOCKER_TOKEN }}
-      
+
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v1
+
+      - name: Set up Docker Buildx
+        id: buildx
+        uses: docker/setup-buildx-action@v1
+        
       - name: Extract metadata (tags, labels) for Docker
         id: meta
         uses: docker/metadata-action@v3
         with:
           images: vstadtmueller/cert-manager-webhook-powerdns
-      
-      - name: Build and push Docker image
+          
+      - name: Build and push docker image
         uses: docker/build-push-action@v2
         with:
           context: .
-          push: true
+          platforms: linux/amd64,linux/arm/v7
```

Comment:
Should be linux/amd64, linux/arm64. Otherwise it will not be 64-bit for arm.

Reply:
I have added it #84